speculos/main.py

```diff
@@ -106,11 +106,20 @@ def get_elf_infos(app_path):
         svc_cx_call_symbol = symtab.get_symbol_by_name("SVC_cx_call")
         if svc_cx_call_symbol is not None:
             svc_cx_call_addr = svc_cx_call_symbol[0]['st_value'] & (~1)
+        # Check where are located fonts in .elf file (LNX/LNS+ only)
+        # (Stax fonts are loaded at a known location: STAX_FONTS_ARRAY_ADDR)
+        fonts_addr = 0
+        fonts_size = 0
+        bagl_fonts_symbol = symtab.get_symbol_by_name('C_bagl_fonts')
+        if bagl_fonts_symbol is not None:
+            fonts_addr = bagl_fonts_symbol[0]['st_value']
+            fonts_size = bagl_fonts_symbol[0]['st_size']
+            print(f"Found C_bagl_fonts at 0x{fonts_addr:X} ({fonts_size} bytes)\n")
```

Comment:
log.info maybe?

Reply:
It was a debug info => I will remove it.